Overview.src.html

```diff
@@ -3026,6 +3080,45 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
  to `<code title>GET</code>` and <var>request</var>'s <span title=concept-request-body>body</span>
  to null.
 
+ <li><p>Let <var>useReferredTokenBinding</var> be the result of
```

Comment:
Since this `<li>` contains multiple block-level children it should be followed by a newline. https://github.com/whatwg/fetch/blob/master/README.md has formatting guidelines that should help.